src/xcode/ENA/ENA/Source/Workers/Keychain/KeychainHelper.swift

```diff
@@ -105,7 +105,7 @@ struct KeychainHelper {
 		}
 
 		let key = "x'\(Data(bytes).hexEncodedString())'"
-		try saveToKeychain(key: ContactDiaryStoreV1.encriptionKeyKey, data: Data(key.utf8))
+		try saveToKeychain(key: ContactDiaryStoreV2.encriptionKeyKey, data: Data(key.utf8))
```

Comment:
I think we can remove the versioning from the store name. With the decoupled schema implementations, we should not need the old store versions anymore.